src/test/java/io/vlingo/actors/ConfigurationTest.java

```diff
@@ -163,12 +173,34 @@ public void testThatConfigurationDefaults() {
     assertEquals("overrideSupervisor", configuration.defaultSupervisorOverridePluginConfiguration().name(0));
     assertEquals(DefaultSupervisorOverride.class, configuration.defaultSupervisorOverridePluginConfiguration().supervisorClass(0));
 
+    assertNotNull(configuration.directoryEvictionConfiguration());
+    assertEquals("directoryEviction", configuration.directoryEvictionConfiguration().name());
+    assertFalse(configuration.directoryEvictionConfiguration().isEnabled());
+    assertEquals(600000, configuration.directoryEvictionConfiguration().lruThresholdMillis());
+    assertEquals(0.8F, configuration.directoryEvictionConfiguration().fillRatioHigh(), 0);
+
     assertEquals("target/classes/", configuration.mainProxyGeneratedClassesPath());
     assertEquals("target/generated-sources/", configuration.mainProxyGeneratedSourcesPath());
     assertEquals("target/test-classes/", configuration.testProxyGeneratedClassesPath());
     assertEquals("target/generated-test-sources/", configuration.testProxyGeneratedSourcesPath());
   }
 
+  @Test
+  public void testThatConfigurationIsLoadedFromFile() {
+    Properties properties = new Properties();
+    Configuration configuration = Configuration.defineWith(properties.properties);
+
+
+
+    configuration.load(Integer.MAX_VALUE);
+
+    assertNotNull(configuration.directoryEvictionConfiguration());
```

Comment:
@VaughnVernon this is where the test fails. I have tried a few different ways to instantiate the Configuration but nothing seems to work.